sandworm/lib/rclone.ml

```diff
@@ -0,0 +1,9 @@
+let copy ~config_path src dest =
+  let cmd = "rclone" in
+  let args = [ "copy"; "-v"; "--config"; config_path; src; dest ] in
+  let cmd = Filename.quote_command cmd args in
+  let ret = Sys.command cmd in
```

Comment:
I'd usually prefer not to use the shell and avoid `Sys.command` as that will avoid the need for quoting (and weird cases if people use weird shells) and you get to control stdout, stderr and return codes better if you use `Unix.{create,open}_process`.